### PR TITLE
SupercheQ

### PR DESCRIPTION
**Pull request by singular-value**
@pgoiporia @vtomole to take over

### CODE REVIEW

File: examples/SupercheQ.ipynb [hunk 1]
```diff
@@ -0,0 +1,407 @@
+{
```
Comment (vtomole) [inline examples/SupercheQ.ipynb:1]: <p>What's this graph saying? The more qubits, the lower the fidelity?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/qiskit-superstaq/pull/276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

**Comment by singular-value** [comment]
Yes. This is (max) fidelity between pairs of fingerprints of distinct files, so we want the fidelity to be low.

More qubits = bigger Hilbert space = more room to cram in fingerprints that are far apart from each other.

**Comment by singular-value** [comment]
Errr sorry if not clear but @pgoiporia and @vtomole, this is a super super rough notebook/ my hope is you guys can make it nice

**Comment by singular-value** [comment]
oh maybe you're already on it

**Comment by vtomole** [comment]
@singular-value Given how on crunch this demo is, the best we can do right now is explain what the snippets you wrote are doing. We will make it nice in the following weeks but the demo you will link to for tomorrow will be an MVP.

**Review by vtomole — APPROVED** [review]
LGTM

**Pull request merged**